platform-includes/logs/default-attributes/mobile.mdx

```diff
@@ -0,0 +1,9 @@
+### Mobile and Native Device Attributes
+
+For mobile applications:
+
+- `os.name`: The name of the operating system
+- `os.version`: The version of the operating system
+- `device.brand`: The brand of the device
+- `device.model`: The model of the device
+- `device.family`: The family of the device
```

Comment:
The list on mobile will grow for sure. I would at least add simulator for now cause that's important and all mobile SDKs should send this.

```suggestion
- `device.simulator`: A flag indicating whether this device is a simulator or an actual
device.
```